src/app/core/interfaces/firebase-item.ts

```diff
@@ -0,0 +1,7 @@
+export interface FirebaseItem {
+    name?: string,
+    price?: number,
+    weigth?: number,
```

Comment:
Are all of these params optional?
Does it match our Dish interface or it is for another data?